File: Esame_28/JAVA/Furgone.java

public class Furgone extends Veicolo{
	private String categoria;
	private int numPosti;

	public Furgone(int codiceVeicolo, String tipoVeicolo, String targaVeicolo, String modelloVeicolo,String marcaVeicolo,
			String categoria, int numPosti)
	{
		super(codiceVeicolo, tipoVeicolo, targaVeicolo, modelloVeicolo, marcaVeicolo);
		this.categoria = categoria;
		this.numPosti = numPosti;
	} 
	
	// metodi
	public String toString()
	{
		return super.toString() + "\t-\t-\t" + this.categoria +"\t"+ this.numPosti;
	}
	
}

File: Esame_28/PYTHON/Furgone.py
from Veicolo import Veicolo


class Furgone(Veicolo):
    # COSTRUTTORE
    def __init__(self, codiceVeicolo, tipoVeicolo, targaVeicolo, modelloVeicolo, marcaVeicolo,
                 categoria, numPosti):
        super().__init__(codiceVeicolo, tipoVeicolo, targaVeicolo, modelloVeicolo, marcaVeicolo)
        self._categoria = categoria
        self._numPosti = numPosti

    # TOSTRING PUNTO 3
    def toString(self):
        return super().toString() + "\t-\t-\t" + self._categoria + "\t" + str(self._numPosti)
